components/Signin/Signin.js

```diff
@@ -33,18 +33,18 @@ export default class Signin extends Component {
     this.logout = this.logout.bind(this);
   }
 
-  _sendSignInAlert() {
+  _sendSignInAlert(error) {
     Alert.alert(
       'Oooops',
-      'Looks like there was a problem. Are you already a member? Double check your inputs, and please try your request again.',
```

Comment:
I added the errors to the alerts since we weren't using them otherwise. We can also remove them along with the error params in function definitions.